gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/traverser/LP_O_OB_P_S_SE_SL_Traverser.java

```diff
@@ -106,17 +106,18 @@ public void dropPath() {
 
     @Override
     public int hashCode() {
-        return super.hashCode() + this.path.hashCode();
+        if (carriesUnmergeableSack()) {
```

Comment:
Can you use inline `returns carriesUnmergableSack() ? ... : ...` please?

Reply:
Be glad that your agreeability score is 1. Mine is 3, thus I won't start a discussion about these 2 variants ending up having the same bytecode. Will change that tomorrow.